src/components/map/cluster/MapCluster.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Weird the naming: ClusterMap, MapCluster LOL
Keep the same order -> ClusterMap and ClusterMapXXX ?

Reply:
ClusterMap is the map, mapCluster is the cluster circle

Reply:
[low]Blanc bonnet et bonnet blanc ;-).
Find another meaningful name ;-P